packages/tools/replay-tool/src/helpers.ts

```diff
@@ -116,7 +112,7 @@ export async function loadContainer(
 	documentServiceFactory: IDocumentServiceFactory,
 	documentName: string,
 	strictChannels: boolean,
-	logger?: ITelemetryLogger,
+	logger?: ITelemetryLoggerExt,
```

Comment:
This can/should be `Base`, since it's passed into `new Loader` below as `ILoaderProps.logger` which is of the `Base` type.

Note that a logger of the `Ext` logger type can be passed into a function like this that takes the `Base` logger type.  For a function parameter, better to take in the least-restrictive type (`Base`, in this case), to be more flexible.

Reply:
Make sense. Changed to `Base` type. thx